### PR TITLE
Bc refactor docker

### PR DESCRIPTION
**Pull request by correiabrux**
Refactoring Dockerfile for image resize.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 COPY . /go/src/github.com/luizalabs/teresa
 
 RUN make build-server
 
+FROM debian:9-slim
+RUN apt-get update && \
+apt-get install ca-certificates -y &&\
+rm -rf /var/lib/apt/lists/*
+
+WORKDIR /go/src/github.com/luizalabs/teresa
```
Comment (yagonobre) [inline Dockerfile]: Maybe use `/app` as workdir

Comment (drgarcia1986) [reply Dockerfile]: I prefer to follow the go pattern

Comment (yagonobre) [reply Dockerfile]: IMO it's valid only in builder step

Comment (drgarcia1986) [reply Dockerfile]: Gotcha 👍

**Comment by drgarcia1986** [comment]
@correiabrux could you update the CHANGELOG ?

File: Dockerfile (hunk 1 above)
Comment (aguerra) [reply Dockerfile]: Take a look if the cache dir is /var/lib/apt/lists or /var/cache/apt/archives (maybe both).

**Pull request merged**